public/js/main.js

```diff
@@ -39,7 +39,8 @@
     }
 
     kitten_imagery.setAttribute('class', 'kitten')
-
+    kitten_imagery.setAttribute('href', window.kittens[index].url)
```

Comment:
Ah, I see how it was confusing.

If we do provide content links they will likely have to exist as separate elements since the video tag on mobile Safari requires clicking the element in order to have the content play, which I think would conflict with linking behavior were it wrapped in an anchor tag.

So, we could explore some sort of permalink solution in another PR. 🚀